Mage.Tests/src/test/java/org/mage/test/cards/mana/DoublingCubeTest.java

```diff
@@ -0,0 +1,35 @@
+package org.mage.test.cards.mana;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+import org.mage.test.serverside.base.impl.CardTestAPIImpl;
+
```

Comment:
minor: add a block comment for the card Doubling Cube since it'll be used in all these tests.

```
/*
 Doubling Cube {2}
Artifact
{3}, {T}: Double the amount of each type of mana in your mana pool.
*/